exe-unit/components/gsb-http-proxy/src/counters.rs

```diff
@@ -0,0 +1,183 @@
+use std::sync::{
+    atomic::{AtomicU64, Ordering},
+    Arc, RwLock,
+};
+
+use chrono::{DateTime, Duration, Utc};
+
+use ya_counters::error::CounterError;
+use ya_counters::Result as CountersResult;
+use ya_counters::{Counter, CounterData};
+
+#[derive(Default, Clone, Debug)]
+pub(super) struct Counters {
+    requests: Option<RequestCounter>,
+    requests_duration: Option<SharedRequestsDurationCounter>,
```

Comment:
I did not tried to handle them as some `Vec`, because for now there are only two counters, and there is a difference between them - only `requests_duration` counter cares about response